test/help/__snapshots__/help.test.js.snap.devServer3.webpack5

```diff
@@ -102,20 +102,20 @@ Options:
   --node-env <value>                               Sets process.env.NODE_ENV to the specified value.
   --progress [value]                               Print compilation progress during build.
   -j, --json [value]                               Prints result as JSON or store it in a file.
-  -d, --devtool <value>                            Determine source maps to use.
-  --no-devtool                                     Do not generate source maps.
-  --entry <value...>                               The entry point(s) of your application e.g. ./src/main.js.
-  --mode <value>                                   Defines the mode to pass to webpack.
+  --devtool <value>                                A developer tool to enhance debugging (false | eval | [inline-|hidden-|eval-][nosources-][cheap-[module-]]source-map).
+  --no-devtool                                     Negative 'devtool' option.
+  --entry <value...>                               A module that is loaded upon startup. Only the last one is exported.
+  --mode <value>                                   Enable production optimizations or development hints.
   --name <value>                                   Name of the configuration. Used when loading multiple configurations.
-  -o, --output-path <value>                        Output location of the file generated by webpack e.g. ./dist/.
-  --stats [value]                                  It instructs webpack on how to treat the stats e.g. verbose.
-  --no-stats                                       Disable stats output.
-  -t, --target <value...>                          Sets the build target e.g. node.
+  --output-path <value>                            The output directory as **absolute path** (required).
+  --stats [value]                                  Stats options object or preset name.
+  --no-stats                                       Negative 'stats' option.
+  --target <value...>                              Environment to build for. Environment to build for. An array of environments to build for all of them when possible.
   --no-target                                      Negative 'target' option.
-  -w, --watch                                      Watch for files changes.
```

Comment:
I think keeping short `-w`/`-d` and ethc make sense, they are very popular, let's keep them, what do you think?

Reply:
Yes, we can add `alias` support for some popular flags `devtool (-d)`, `target (-t)`, `--output-path (-o)`, `--watch (-w)`

Reply:
Ideally, we should add this in the webpack schema itself. 
Or we can write logic to generate an alias property for each option in webpack-cli. 
Or add some workaround logic to support alias for some popular options only (coming from webpack core).

Reply:
> Or we can write logic to generate an alias property for each option in webpack-cli.

I think so, it was my original idea, so we will support all options using short names, we can do it separatly

Reply:
Yes. Let's add them separately. In the meantime should I update the PR to keep the popular ones?

Reply:
Yes, let's keep popular options here, some of them we can kept for better compatibility

Reply:
@alexander-akait Added in https://github.com/webpack/webpack-cli/pull/3342/commits/00fae2fca6baa314b46e6207fd9bc54b141cd5f9